engine/src/main/java/org/terasology/engine/core/TerasologyEngine.java

```diff
@@ -266,7 +266,7 @@ private void verifyInitialisation() {
      * Logs software, environment and hardware information.
      */
     private void logEnvironmentInfo() {
-        logger.info(TerasologyVersion.getInstance().toString());
+        logger.info("{}", TerasologyVersion.getInstance().toString());
```

Comment:
```suggestion
        logger.info("{}", TerasologyVersion.getInstance());
```

Reply:
Interestingly, checkstyle didn't suggest to remove `toString()` here while in other places it did. I don't fully understand yet why it does it in some cases and doesn't in others. Any ideas?

Reply:
Technically, you can `toString` on something that is already a `String` - maybe Checkstyle understands this and tells you that that is superflous? 🤔

Reply:
Oh, that's a good point... so it tells me because `toString()` is not necessary on `Strings` but not because `toString()` is not necessary as it's implicitly called when logging 💡 makes sense 👍